src/did/web5-did.js

```diff
@@ -1,19 +1,20 @@
 import { Encoder } from '@tbd54566975/dwn-sdk-js';
 
-import { DIDConnect } from './connect/connect.js';
+import { DidConnect } from './connect/connect.js';
 import * as CryptoCiphers from './crypto/ciphers.js';
+import * as DidManager from './manager.js';
 import * as Methods from './methods/methods.js';
-import * as DidUtils from './didUtils.js';
-import { MemoryStorage } from '../storage/MemoryStorage.js';
+import * as DidUtils from './utils.js';
+import { MemoryStorage } from '../storage/memory-storage.js';
 import { pascalToKebabCase } from '../utils.js';
 
-class Web5DID {
+export class Web5Did {
   #cryptoCiphers = {};
   #didConnect;
   #web5;
 
-  #registeredDIDs = new MemoryStorage();
-  #resolvedDIDs = new MemoryStorage();
+  #resolvedDids = new MemoryStorage();
+  didStore = new MemoryStorage();
```

Comment:
should this be private or public?

Reply:
it can be private.  Addressed in https://github.com/TBD54566975/web5-js/pull/43/commits/a01471f36de09561c63155887265985b717fc000